Assets/MRTK/Examples/Common/Scripts/LoadProfilesOnStartup.cs

```diff
@@ -15,17 +15,13 @@ public class LoadProfilesOnStartup : MonoBehaviour
         [SerializeField]
         private MixedRealityToolkitConfigurationProfile configProfile = null;
 
-        private void Start()
-        {
-            MixedRealityToolkit.Instance.ActiveProfile = null;
-        }
-
         private void LateUpdate()
```

Comment:
Should this happen in `Update` instead, since MRTK.cs checks in `LateUpdate` for the presence of a new profile? Otherwise it seems like we might end up not switching until two frames later due to script execution order.